src/components/Reports/ReportsNew.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import DopplerIntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import Reports from './ReportsNew';
+
+describe('Reports New page', () => {
+  afterEach(cleanup);
+
+  it('render page', () => {
+    // Arrange
+    // Act
+    const { getByText } = render(
+      <DopplerIntlProvider>
+        <Reports />
+      </DopplerIntlProvider>,
+    );
+
+    // Assert
+    expect(getByText('Reports new page'));
```

Comment:
What do you think about expect for `reports_title`? 

Because, in other case, `Reports new page` will be modified soon and you will need to update the PR again.